Python3.md

```diff
@@ -0,0 +1,16 @@
+# Python 3 Guide
+_Python 3 for Python 2 devs._
+
+## Super calls
+
+Whenever possible, projects should take advantage of Python 3 features and syntax. For example prefer [new style super calls](https://docs.python.org/3.5/library/functions.html#super) to old stlye.
```

Comment:
"should take advantage of Python 3 features..." doesn't make sense in this context